### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by mperezbazan**
Created media query for Desktop Versions in sections:
Headline
Works
About
Contact
Change images to fit news sizes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,42 +38,52 @@ <h1 class="headline-title">Hey There.<br>I’m Martin</h1>
...
+      </article>
+      <article class="work-container work-light sm-none">
+        <div class="work-footer-container-dark">
+          <a class="work-button button dark fullwidth">
             See Project
           </a>
         </div>
       </article>
-      <article class="work-container bg-image">
+      <article class="work-container bg-image data-image">
```
Comment (tresorsawasawa) [inline index.html:79]: - [ ] Kindly make sure the spacing between both cards sticks to the one from the Figma design.

#### Figma Design 
Screenshot from 2022-09-29 18-34-18

#### Your Design 
Screenshot from 2022-09-29 18-30-24

File: index.html [hunk 2]
```diff
@@ -263,7 +292,7 @@ <h3 id="form-headline">
         <li class="footer-social-media-button"><a href="https://twitter.com/mperezbazan"><img src="./images/Twitter-dark.svg" alt="Twitter icon"/></a></li>
         <li class="footer-social-media-button"><a href="https://medium.com/@martin.perezbazan"><img src="./images/Medium-dark.svg" alt="Medium icon"/></a></li>
       </ul>
-      <hr class="footer-bottom-line"/>
```
Comment (tresorsawasawa) [inline index.html:266]: - [ ] Kindly make sure all the bottom social media icons are reading perfectly as right now some of them are broken to make the project more professional
Screenshot from 2022-09-29 18-37-56

File: css/style.css [hunk 3]
```diff
@@ -388,3 +424,213 @@ hr.works-divider {
...
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .work-text.light {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .data-image {
```
Comment (tresorsawasawa) [inline css/style.css]: - [ ] Kindly make sure the **Hamburger** button is visible and **all the navigations options(e.g.: Portfolio, Contact,...)** are hidden on the **"Mobile Version"**. And on the Desktop version, the **Hamburger** button should be hidden, and **all the navigation options** are visible. Right now, your navigations options are visible for both the **Mobile Version** and **Desktop Vesrion**

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @mperezbazan 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -263,7 +292,7 @@ <h3 id="form-headline">
         <li class="footer-social-media-button"><a href="https://twitter.com/mperezbazan"><img src="./images/Twitter-dark.svg" alt="Twitter icon"/></a></li>
         <li class="footer-social-media-button"><a href="https://medium.com/@martin.perezbazan"><img src="./images/Medium-dark.svg" alt="Medium icon"/></a></li>
```
Comment (youmari) [inline index.html:293]: - [ ] Great job making the requested changes but you forgot to make sure that your social media icons are displaying properly so please try to fix that 
Screen Shot 2022-09-29 at 18 42 56

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @mperezbazan , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Github flow 
✔️ PRO PR
✔️ No linter error 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (mperezbazan) [reply index.html:293]: @youmari Thanks for your feedback, I checked it on my PC, and my coding partner checked on his PC and it's working well, all the social media icons were in the place
image

File: index.html [hunk 5]
```diff
@@ -257,13 +286,13 @@ <h3 id="form-headline">
     <!-- Footer -->
     <section class="footer-container">
       <ul class="footer-button-group">
-        <li class="footer-social-media-button"><a href="https://github.com/mperezbazan"><img src="./images/GitHub-dark.svg" alt="Github icon"/></a></li>
+        <li class="footer-social-media-button"><a href="https://github.com/mperezbazan"><img src="./images/GitHubDark.svg" alt="Github icon"/></a></li>
```
Comment (MaganaAsiati) [inline index.html]: - [ ] Kindly make sure you use the correct path for your GitHub icon as required by the previous reviewer . On my side, it is still showing the broken link for the image, and the image is not displayed as expected. check the picture below of how it looks on my side :+1: 

## Your design 

image

## Figma design

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @mperezbazan , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] [OPTIONAL] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by jerryowusu — APPROVED** [review]
Hi @mperezbazan :wave:  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Well done for implementing the changes requested by the previous reviewer. :+1: 
Congratulations! 🎉 

good job


## Highlights
:heavy_check_mark: Followed GitHub Flow
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: All Linters are passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**